cluster-scope/base/core/namespaces/stackrox/kustomization.yaml

```diff
@@ -1,7 +1,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
-components:
-- ../../../../components/project-admin-rolebindings/stackrox-admin
 kind: Kustomization
 namespace: stackrox
 resources:
 - namespace.yaml
```

Comment:
@traytorous while at it, can you add some annotations to the namespace?
Like in https://github.com/operate-first/apps/blob/master/cluster-scope/base/core/namespaces/b4mad-racing/namespace.yaml